src/hotspot/cpu/aarch64/macroAssembler_aarch64.hpp

```diff
@@ -106,7 +106,6 @@ class MacroAssembler: public Assembler {
   static const char* describe_klass_decode_mode(KlassDecodeMode mode);
 
  private:
-
```

Comment:
Missed a similar cleanup in `macroAssembler_x86.hpp`, I think.

Reply:
Right. I removed the newline in that file, too.